simcon_project/simcon_project/urls.py

```diff
@@ -15,8 +15,7 @@
 """
 from django.contrib import admin
 from django.urls import path
-from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, 
-  UpdateOverallResponseFeedback, UpdateNodeResponseFeedback, SettingsView, TemplateStartView
+from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, UpdateOverallResponseFeedback, UpdateNodeResponseFeedback, ResearcherSettingsView, TemplateStartView, ResearcherUserView, StudentSettingsView,StudentUserView
```

Comment:
I'm thinking we might use `import *` here since this line is getting pretty disgusting. OR line continuation if that can be done with imports.

Reply:
yeah i wasnt sure how to clean this up, lmk what u think